### PR TITLE
Change event announcement text "station" --> "shuttle"

### PR DESCRIPTION
**Pull request by Mygnol**
## About the PR

Changes references of "the station" to "the shuttle" for common events.

Right now I've just commented out the old text and replaced it with the new one; let me know if creating new files in ``_NF/events`` and putting everything there instead would be preferred.

## Why / Balance

Changes station event announcements to clear up potential new player confusion.

## How to test

Addgamerule the changed events.

## Media

image
image
image
image
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

- The PowerGridCheck voiceline no longer matches 100% with the actual text :<

**Changelog**

Trivial change, no CL

### CODE REVIEW

**Comment by Mygnol** [comment]
> Prefer plurals to a singular shuttle.

Edited.

**Comment by dvir001** [comment]
Ill probably end up moving all our events into a "NF" versions later so doing anything on _NF for it will be great

**Comment by whatston3** [comment]
Put up a set of suggestions at [whatstone:2024-11-28-bluespace-announcement-suggestions (vs. master)](https://github.com/new-frontiers-14/frontier-station-14/compare/master...whatston3:2024-11-28-bluespace-announcement-rename-suggestions?expand=1)

Changes seem fine on the whole - I'd prefer a generic "shuttles" to "some shuttles", and if adding frontier-specific strings, keep the existing ones as they are and change the string being used in the event itself.

Gave a small set of changes to other items that could be more Frontier-specific.

Think this is good to go with those changes, happy to work out the differences.

**Comment by Mygnol** [comment]
> Put up a set of suggestions at [whatstone:2024-11-28-bluespace-announcement-suggestions (vs. master)](https://github.com/new-frontiers-14/frontier-station-14/compare/master...whatston3:2024-11-28-bluespace-announcement-rename-suggestions?expand=1)
> 
> Changes seem fine on the whole - I'd prefer a generic "shuttles" to "some shuttles", and if adding frontier-specific strings, keep the existing ones as they are and change the string being used in the event itself.
> 
> Gave a small set of changes to other items that could be more Frontier-specific.
> 
> Think this is good to go with those changes, happy to work out the differences.

Looks sane enough to me. I originally considered your method of changing the strings on the event as well, my implementation was more out of concern for whether downstream would get their stuff un-translated. 

Shuttles vs "some shuttles" was really just because it flows nicer. No real preferences there.

File: Resources/Locale/en-US/_NF/events/gas-leak.ftl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+﻿station-event-gas-leak-nf-start-announcement = Attention. Gas leaks have been detected on shuttles. Affected crew are advised to avoid affected areas and wear suit internals in the meantime.
+station-event-gas-leak-nf-end-announcement = The source of the gas leaks have been fixed. Please be cautious around areas with gas remaining.
```
Comment (whatston3) [inline Resources/Locale/en-US/_NF/events/gas-leak.ftl:2]:
```suggestion
station-event-gas-leak-nf-end-announcement = The source of the gas leaks has been fixed. Please be cautious around areas with gas remaining.
```

Comment (whatston3) [reply Resources/Locale/en-US/_NF/events/gas-leak.ftl:2]: My mistake.

File: Resources/Locale/en-US/station-events/events/ion-storm.ftl [hunk 2]
```diff
@@ -17,7 +19,7 @@ ion-storm-x-and-y = {$x} AND {$y}
 # joined is short for {$number} {$adjective}
 # subjects can generally be threats or jobs or objects
 # thing is specified above it
-ion-storm-law-on-station = THERE ARE {$joined} {$subjects} ON THE STATION
+ion-storm-law-on-station = THERE ARE {$joined} {$subjects} IN THE SECTOR
```
Comment (whatston3) [inline Resources/Locale/en-US/station-events/events/ion-storm.ftl:22]:
```suggestion
# Frontier: "on the station" < "in the sector"
ion-storm-law-on-station = THERE ARE {$joined} {$subjects} IN THE SECTOR
```

**Review by whatston3 — APPROVED** [review]
Fixed a few remaining issues.

**Pull request merged**